test/integration/blob-tests.sh

```diff
@@ -38,14 +38,17 @@ testing class "listblobs"
 BC=5 # blob count
 test "listblobs basic" assert.Success "azmi listblobs --container $CONTAINER_LB"
 test "listblobs finds $BC blobs" assert.Equals "azmi listblobs --container $CONTAINER_LB | wc -l" $BC
+test "listblobs finds $BC blobs (absolute-paths flag enabled)" assert.Equals "azmi listblobs --container $CONTAINER_LB --absolute-paths | wc -l" $BC
```

Comment:
I think it would be good to add some check which will verify if return value is really absolute path. Something like `| grep https` or somthing similar. What do you think?

Maybe similar also for command without `--absolute-path`, where `| grep https` should fail?

Reply:
I had same idea, but somehow let it go :) You reminded that to me.
Added.